style.css

```diff
@@ -465,91 +469,85 @@ button {
   align-items: center;
   text-align: center;
   color: #fff;
-  margin-top: 10rem;
+  margin-top: 6rem;
 }
 
 .contact-me-text {
   width: 327px;
   height: 137px;
   left: 24px;
-  top: 4802px;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 400;
   font-size: 20px;
   line-height: 28px;
   text-align: center;
   color: #ebebff;
+  margin-top: -1rem;
 }
 
-.input-box {
-  margin: 1em 2em 0 1em;
+.contact-form {
   display: flex;
-  flex-direction: column;
-  align-self: flex-start;
+  flex-flow: column;
+  align-items: flex-start;
+  gap: 10px;
   width: 90%;
+  margin-left: 1rem;
+  margin-right: 1rem;
+  margin-top: 2.5rem;
+}
+
+.input-box {
+  width: 100%;
   height: 3rem;
   background: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-size: large;
   border-radius: 1em;
+  border: 1px solid #cfd8dc;
 }
 
-.input-text {
-  display: flex;
-  flex-direction: column;
-  padding-left: 1em;
-  justify-content: center;
-  font-size: medium;
+input[type='text'] {
+  padding-left: 10px;
+  max-width: calc(100% - 10px);
+  color: black;
 }
 
-textarea {
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  margin-top: 1em;
-  height: 60%;
-  display: flex;
-  align-self: flex-start;
-  position: relative;
-  left: -6.5em;
+input[type='email'] {
+  padding-left: 10px;
+  max-width: calc(100% - 10px);
+  color: black;
+  font-size: large;
 }
 
 .message-box {
-  margin: 1em 2em 0 1em;
-  display: flex;
-  flex-direction: column;
-  align-self: flex-start;
-  width: 90%;
-  height: 10rem;
+  margin: auto;
+  width: 100%;
+  height: 8rem;
   background: #fff;
   border-radius: 1em;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  color: #b3bac5;
+  font-size: large;
+  color: black;
+  padding-left: 10px;
+  padding-top: 10px;
+  max-width: calc(100% - 10px);
 }
 
 .getintouch {
   background-color: white;
-  width: 40%;
-  height: 2rem;
+  height: 3rem;
   border-radius: 0.8em;
-  margin-bottom: 1rem;
-  padding: 1.5em 1em 2em 0.3em;
   border: 1px solid;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 1em;
-  line-height: 0.2rem;
   text-align: center;
```

Comment:
Please make sure your design is the same as Figma's design according to the project requirement. It will give a good appearance to your portfolio.  Currently, the `border-radius`  and the `margin` of your button are not the same as the Figma design. Kindly, fix it.
![image](https://github.com/AliRezaBmeDu/Portfolio-Website/assets/101523582/3feed73a-e525-43b6-b521-76e5c094dff4)

Figma design
![image](https://github.com/AliRezaBmeDu/Portfolio-Website/assets/101523582/331458da-bfb9-40bd-ab2a-3a2247cb3552)